assembly.html.markdown

```diff
@@ -0,0 +1,218 @@
+---
+language: assembly
+filename: learn.asm
+contributors:
+    - ["", ""]
+---
+
+Assembly language is a low-level programming language for a computer
+
+```asm
+; Comments
```

Comment:
A little more verbose here. As a novice, I'm not sure if this is special syntax, whether "Comments" has special meaning like "section" does. Maybe "Anything from semicolon until the end are comments"?